packages/web-app-search/src/types.ts

```diff
@@ -0,0 +1,31 @@
+export interface SearchResult {
```

Comment:
TODO: Document this as minimal requirements to implement a custom search yourself

Reply:
to get this merged we skip that for now, the interface is a good start and the actual implementations are there as a bluePrint.